src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,370 @@
+in voice chat, eventually setting up a game on [skribbl.io](https://skribbl.io).
+
+As part of their turn, one of the players was tasked with drawing a pumpkin. This lead to many people typing in
+"pineapple" as their guess -- and thus, Pineapple was born!
+
+<figure>
+<a href="/assets/img/pineapple.png">
+<img alt="Pineapple, one of Quilt's mascots" src="/assets/img/pineapple.png" />
+</a>
+</figure>
+
+Pineapple is a core part of our community's culture, and we often use it to represent our community -- both officially
+and informally. If you find yourself in Quilt community spaces, you'll definitely see many variants of Pineapple's
+original drawing -- please feel free to create your own art too, if you'd like!
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}What's up with Quilt's community moderation?{% endsummary %}
+
+Among other things, Quilt exists to address issues with poor moderation in some other parts of the Minecraft community.
+We employ an approach that we call "Defensive Moderation" -- an emerging concept that focuses primarily on harm 
+reduction and investigative moderation, keeping an eye on behaviours and trends in the wider community and acting upon
+them to prevent issues before they happen.
+
+The reason our approach feels so different to other communities' is because we investigate and act upon issues that
+appear in the wider community, even if they don't directly affect Quilt. We believe that nothing happens in a vacuum,
+and a problematic situation somewhere else is good evidence that the people involve may cause similar situations in our
+spaces. We're not shy about banning problem users before they even set foot in Quilt community spaces, and some people
+find this perplexing -- but we believe in this approach, and it's been a big contributor in keeping Quilt users safe
+from discrimination and other issues.
+
+Similarly, Quilt itself doesn't operate in a vacuum - we operate a program known as Quilt Community Collab, where we
+work with other related communities on moderation-related issues and events. Working alongside organisations such as
+Forge, Geyser and Paper means that we can contribute to improving the _entire_ community sphere, for everyone -- and
+in return, those organisations help us with our investigations and moderation-related issues.
+
+We believe that all communities can contribute positively to the well-being of their users, and that they have a
+responsibility to create safe, inclusive spaces, and to set a trend doing so -- encouraging more communities to do the
+same, and making the Internet a better place for everyone.
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}Does Quilt send my mods to the servers I play on?{% endsummary %}
+
+This is a misconception based on poor communication: at one point, to help ease the moderation burden in Quilt's
```

Comment:
Was this really primarily intended to be used by Quilt's official spaces?

Reply:
Yep